packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.js

```diff
@@ -49,12 +49,11 @@ export default class QuanticBreadcrumbManager extends LightningElement {
   }
 
   renderedCallback() {
-    initializeWithHeadless(this, this.engineId, this.initialize.bind(this));
+    initializeWithHeadless(this, this.engineId, this.initialize);
```

Comment:
You should use an arrow to capture this if you don't use bind 😄 
```suggestion
    initializeWithHeadless(this, this.engineId, () => this.initialize);
```

Reply:
nvm just realized initialize is already a scoped lambda.